features/steps/sample_steps.py

```diff
@@ -1,10 +1,10 @@
 import json
 import time
+from google.cloud import storage
```

Comment:
Just being a bit picky but google cloud should be back where it was before. It's a third party import so shouldn't be up top with the standard library imports